README.md

```diff
@@ -23,21 +28,25 @@ chmod +x statistics/update.sh
 ```
 
 To have the app always running you will now create two systemd services:
+
 - `mongod.service` - database
 - `node-course-statistics.service` - node.js
 
 This can be done by using [this configuration for mongod](https://gist.github.com/jwilm/5842956) and following [this guide for node](https://www.axllent.org/docs/view/nodejs-service-with-systemd/). Some tweaks might be neccesary to get it working for this specific app. For example the node service should start `node server.js` in our case.
 
-To get continuous database updates you need to setup a cron job.
-Here is an example that will run `update.sh` every second day.:
+**NOTE** The data is currently obtained by manual email as Chalmers does not publish it as a online document anymore. The paragraph below is the old way of updating the data which might be relevant again in the future. 
 
-`0 0 1-31/2 * * /srv/websites/statistics/statistics/update.sh /srv/websites/statistics/statistics >/dev/null 2>&1`
+_To get continuous database updates you need to setup a cron job.
+Here is an example that will run `update.sh` every second day.:_ 
```

Comment:
Has the cron job been disabled?

Reply:
I've contacted Spidera about the server end of things as I don't have access to it anymore.

Reply:
Yes, the cron job has been disabled.